.github/ISSUE_TEMPLATE/release.yaml

```diff
@@ -51,13 +51,14 @@ body:
   attributes:
     label: "**For major/minor releases only** Update Release documents"
     options:
-      - label: Trigger [release_docs](https://github.com/Kong/kubernetes-ingress-controller/blob/main/.github/workflows/release_docs.yaml) workflow. Note that you will need to update the new version's navigation manifest (e.g. [for 2.7](https://github.com/Kong/docs.konghq.com/blob/main/app/_data/docs_nav_kic_2.7.x.yml) to use the new files (CRDs and CLI arguments references) after.
+      - label: Trigger [release_docs](https://github.com/Kong/kubernetes-ingress-controller/blob/main/.github/workflows/release_docs.yaml) workflow.
       - label: Ensure a draft PR is created in [docs.konghq.com](https://github.com/Kong/docs.konghq.com/pulls) repository.
       - label: If you are adding a new CRD, add a new description file under `app/_includes/md/kic/crd-ref/`. This is a brief description injected into the CRD reference page.
       - label: Update articles in the new version as needed.
       - label: Update `reference/version-compatibility.md` to include the new versions (make sure you capture any new Kubernetes/Istio versions that have been tested)
       - label: Copy `app/_data/docs_nav_kic_OLDVERSION.yml` to `app/_data/docs_nav_kic_NEWVERSION.yml` and update the `release` field to `NEWVERSION`. Add entries for any new articles.
-      - label: Make sure that `app/_data/docs_nav_kic_NEWVERSION.yml` links to the latest generated `custom-resources-X.X.X.md`.
+      - label: Make sure that `app/_data/docs_nav_kic_NEWVERSION.yml` custom resources link to the latest generated `custom-resources-X.X.X.md`.
+      - label: Make sure that `app/_data/docs_nav_kic_NEWVERSION.yml` cli arguments link to the latest generated `cli-arguments-X.X.X.md`.
```

Comment:
You'll need to copy the files as well.

Reply:
In https://github.com/Kong/docs.konghq.com/pull/8305 we needed to manually copy the files because we were adding the nav for 3.5 upfront (as unreleased). I don't think this is something good to do, as the cli args and the custom-resources files were automatically generated by the bot in https://github.com/Kong/docs.konghq.com/pull/8298.

Reply:
I've addressed it in 1784b398f823ba5f6ee5e258360ce069c3fac61f. PTAL